packages/docs/blog/2021-11-17-welcome/index.mdx

```diff
@@ -30,14 +30,9 @@ If done right, there will be a long tail representing millions of social tokens
 
 Using Strata, creators can create, manage and build social tokens in just a few clicks. These tokens can be bound to SOL, USDC, or a collective. 
 
-**Collectives** are for like-minded creators that want to pool resources and bond their directional success to others. 
-Within collectives, every token holder is bound to the collective and incentive-aligned to increase not only the value of their token, but the value of the collective as a whole. 
-Imagine a group of artists forming a collective for their genre and pooling resources to create a custom marketplace. 
-Imagine a Decentralized Autonomous Organization (DAO) with a token and a collective of contributors with tokens bound to the DAO token.
-Imagine the ability to bootstrap an entire economy around an idea.
-
-Collectives also allow fans to take directional positions on the success of groups or categories rather than individuals alone.
```

Comment:
I think we still want this bit. That you can take a directional position

Reply:
I believe I moved this else where and changed the wording